laravel/resources/js/admin.js

```diff
@@ -118,11 +118,46 @@ $(document).ready(function()
         $(this).parents('.volunteer').find('.buttons').attr('style', false);
     });
 
+    //volunteer review page
+    $('.volunteer-status-review').on('change',function()
+    {
+        var status = $(this).parents('.volunteer').find('.volunteer-status-review').value();
+        var csrf = $(this).parents('.volunteer').find('.csrf-token').value();
+        var slot = $(this).parents('.volunteer').find('.slot-number').value();
+        var data =
+        {
+            status: status,
+            _token: csrf
+        };
+
+        ajaxOptions.body = JSON.stringify(data);
+        fetch('/slot/'+ slot +'/edit', ajaxOptions);
+
+        var message = $(this).parents('.volunteer').find('.toast-message').style({visibility: 'visible', opacity: 1});
+        setTimeout(function()
+        {
+            clearTimeout()
```

Comment:
This doesn't actually do anything. `clearTimeout` requires an argument (a reference to the timeout being cleared)

Reply:
alright i fixed it to where it actually does that, how silly of me to miss that